docs/website/src/pages/index.js

```diff
@@ -1,34 +1,11 @@
 import React from 'react';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import Layout from '@theme/Layout';
-import Logo from '../../static/img/docusaurus.png'
+import { Redirect } from '@docusaurus/router';
+
 
 export default function Home() {
   const { siteConfig } = useDocusaurusContext();
 
-  return (
-    <Layout
-    title={`Hello from ${siteConfig.title}`}
-    description="Description will go into a meta tag in <head />"
-  >
-    <main>
-      <div class="container text--center">
-        <div class="row">
-          <div class="col">
-            <img class="featureImage_yA8i" alt="Powered by MDX" src="/img/undraw_typewriter.svg" loading="lazy" width="640" height="480"></img>
```

Comment:
- Prefer using `className` instead of `class`
- Remove the "witdth" and "height" from inline style and implement these values as `style` object using `style={styleObject}` or external css sources and use responsive values instead of pixels

Reply:
Thanks for the feedback, @EternalQuasar0206 !

About the style: what option do you think that is more suitable? I researched some options:

- SASS/SCSS
- CSS Modules
- CSS-in-JS

I think that Modules is great for what I've seen, but I'm in doubt now.

Reply:
I believe that SASS is an interesting and viable choice, both to learn and to implement. Its use is very simple and it improves the readability and maintainability of the styles. If you don't think it's feasible, you can use pure CSS without any problems. I recommend trying to avoid CSS-IN-JS as this can create code smell and unreadable and large codes, an alternative to CSS-IN-JS is to use style libraries (like Chakra or MUI)